### ISSUE TITLE
CONTENT: Create a bio (and pick out a name) for the design mascot, a llama

### ISSUE THREAD

**Issue by renrutnnej** [hoodiehq/editorial#66]
Part of https://github.com/hoodiehq/hood.ie/issues/234

### :octocat: Starter Issue

We would like to take the opportunity to invite someone who did not yet contribute to Hoodie to work on this issue. Follow [@YourFirstPR](https://twitter.com/yourfirstPR) on Twitter for more of these issues across many open source projects :relaxed:

---

### :pencil2: The Task

In preparation for the Camp Release we are trying to update the hood.ie site. This specific writing task involves adding content to the [Hoodie Animals page](http://hood.ie/animals/) to give readers a better idea of our revolving cast of animal characters. :dog: 

- [ ] Claim this issue (`comment below or assign yourself`)
- [ ] Write a bio between 200-400 words long.
- [ ] Submit to @renrutnnej @Charlotteis or @janl for review (Google docs probably work best for this sort of thing)
- [ ] You are done :clap: comment below that your pull request is ready for review :tada:

Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/) if you have any questions :)

**Comment by gr2m** [comment]
It's an Alpaca, very important :tm: :)

**Comment by summerspirit** [comment]
No, they're not at all. Alpacas make the softest softest teddy bears and blankets ever <3
Alpaca fur teddy:

![alpaca-teddy-bears-08](https://cloud.githubusercontent.com/assets/4708646/13476872/ac552cac-e0c1-11e5-9398-cbe9b4dae25d.jpg)

(don't worry - they shear it off like we do with sheep)

**Comment by matsieftw** [comment]
I'd love to write a bio for an alpaca!!

**Comment by renrutnnej** [comment]
Thanks @matsieftw, I'll assign you to the issue! Let me know if there's anything I can do to help get you started :relaxed:

**Comment by renrutnnej** [comment]
Hey @matsieftw I just wanted to check in with you to see how you're doing with the Alpaca bio :relaxed:

**Comment by matsieftw** [comment]
@renrutnnej Hey there! Sorry this took me a day to respond. I have a rough draft that I did and am returning to later this afternoon.

**Comment by matsieftw** [comment]
Hey y'all. I just finished my draft for the design alpaca.

https://docs.google.com/document/d/1XNuI-u02OKl3jmoiAwhi3fDBWcvM2WPJ_tucJVWp2Us/edit?usp=sharing

**Comment by espy** [comment]
Lovely :) Thanks a lot for that. Made a few small suggestions, if that's alright. 

Apologies in advance: I'm still totally overthinking the name, wondering if we should allude to some famous painter or designer or someone with a connection to the Andes. Thinking of László (as in [Moholy-Nagy](https://www.google.de/search?hl=de&q=moholy+nagy&ie=utf-8&tbm=isch&gws_rd=cr&ei=EccLV4WeIYb7O821pbgK) ) or Alexander von Humboldt, who explored the Andes. On the other hand, those are two white male Europeans, so that's a bit dull. Alessandro, maybe? Dunno. 

Anway, feel free to disregard that whole paragraph :) You invented this Alpaca, you get to name him!

**Comment by renrutnnej** [comment]
I'm working on getting this bio onto the editorial calendar so we can release it @matsieftw ☺️

**Comment by matsieftw** [comment]
Oh wow. Great. Let me know if there's anything I can do.
